src/router/index.js

```diff
@@ -88,6 +89,12 @@ const routes = [
     component: Products,
     meta: { title: "Products page", requiresAuth: true },
   },
+  {
+    path: "/products/:productId/transfer",
+    name: "Product Transfer",
+    component: ProductTransfer,
+    meta: { title: "Product Transfer", requiresAuth: true },
+  },
```

Comment:
We don't use a separate view for the transfer, right? If that's the case, we should remove this.

Reply:
removed